scholarx/archive/functions.js

```diff
@@ -2,6 +2,10 @@ $(function () {
     loadNavAndFooter('/assets/content/static');  //relative path to content directory
 });
 
+//global variables to hold mentor and mentee data
```

Comment:
So `var` is a global variable right? You put the comment as `global variables to hold mentor and mentee data` bcs of the `var` variable no? So should it change to `variables to hold mentor and mentee data` @janithlahirukariyawasam 
```suggestion
//variables to hold mentor and mentee data
```

Reply:
> So `var` is a global variable right? You put the comment as `global variables to hold mentor and mentee data` bcs of the `var` variable no? So should it change to `variables to hold mentor and mentee data` @janithlahirukariyawasam

Thanks

Reply:
@YoshithaRathnayake It's recommended not to use `var`. For this use case `let` is more suited and It will still act as a global variable. Please correct me if I'm wrong @anjula-sack.